.github/integration/scripts/deploy_charts.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+if [ "$1" == "true" ]; then
+    echo "TLS enabled"
+    sed -i 's/# .*\(ticket_server_tls.key.*\)/\1/' .github/integration/scripts/values.yaml
+    sed -i 's/# .*\(ticket_server_tls.cert.*\)/\1/' .github/integration/scripts/values.yaml
+else
+    echo "TLS disabled"
+    sed -i 's/\(ticket_server_tls.key.*\)/# \1/' .github/integration/scripts/values.yaml
+    sed -i 's/\(ticket_server_tls.cert.*\)/# \1/' .github/integration/scripts/values.yaml
+fi
```

Comment:
```suggestion
if [ "$1" == "false" ]; then
    echo "TLS disabled"
    sed -i '/ticket_server_tls.*/d' .github/integration/scripts/values.yaml
fi
```

TLS is the default value so you only need to remove it if it is disabled.

Reply:
This is handy when running local tests for development, I'd prefer to keep it here.